i18n-tasks.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   TEXT
   s.homepage = 'https://github.com/glebm/i18n-tasks'
   s.metadata = {
+    'changelog_uri' => 'https://github.com/glebm/i18n-tasks/blob/main/CHANGES.md',
     'issue_tracker' => 'https://github.com/glebm/i18n-tasks',
```

Comment:
Could you update it based on the documentation as well?
```suggestion
    'bug_tracker_uri' => 'https://github.com/glebm/i18n-tasks',
```

Reply:
And maybe we can add `source_code_uri` to point to the repo as well?

Reply:
👍 Done in [9732435](https://github.com/glebm/i18n-tasks/pull/583/commits/97324354474c1638214a4dc97bc8ecb881f267c7)